docs/documentation/models/ollama-generation.md

```diff
@@ -0,0 +1,86 @@
+# @axflow/models/ollama/generation
+
+Interface with [local ollama models](https://ollama.ai) using this module.
+
+```ts
+import { OllamaGeneration } from '@axflow/models/ollama/generation';
+import type { OllamaGenerationTypes } from '@axflow/models/ollama/generation';
+```
+
+```ts
+declare class OllamaGeneration {
+  static stream: typeof stream;
+  static streamBytes: typeof streamBytes;
+  static streamTokens: typeof streamTokens;
+}
+```
+
+_Note: Ollama only provides streaming APIs, so Axflow does the same._
```

Comment:
Might be nice to use one of the vitepress callouts here, like `tip` or `info`.

Reply:
<img width="791" alt="CleanShot 2023-09-18 at 17 28 50@2x" src="https://github.com/axflow/axflow/assets/1666947/e642b2bc-81b4-40b6-bcc1-3c652492e497">